app/views/estimate_flow/applicant.html.slim

```diff
@@ -18,35 +18,48 @@
                                           yes_no_options, :first, :last,
                                           legend: { text: t(".over_60.legend") },
                                           hint: { text: t(".over_60.hint") }
-
-    = form.govuk_collection_radio_buttons :dependants,
-            yes_no_options,
-            :first,
-            :last,
-            legend: { text: t(".dependants.legend") },
-            hint: lambda { \
-                            tag.p(t(".dependants.hint")) + \
-                            tag.ul(class: "govuk-list govuk-list--bullet") do \
-                            tag.li(t(".dependants.lives_with")) + \
-                            tag.li(t(".dependants.depends")) \
-                            end + \
-                            tag.div(t(".dependants.no_partner"), class: "govuk-inset-text") \
-                            }
+    - if Flipper.enabled?(:partner)
+      = form.govuk_collection_radio_buttons :dependants,
```

Comment:
Just to warn you that in https://github.com/ministryofjustice/laa-estimate-financial-eligibility-for-legal-aid/pull/236 this field is moving to a different screen, so brace yourself for a merge conflict if that gets merged first.

Reply:
👍 🏎️